pages/global-styles.mdx

```diff
@@ -0,0 +1,87 @@
+# Global styles
+
+Create a file called global.ts within the styles folder and paste the following code. Under GlobalStyle, general style rules are defined for the entire project.
+
+```
+import { createGlobalStyle } from 'styled-components';
+import { normalize } from 'styled-normalize';
+
+const GlobalStyle = createGlobalStyle`
+  ${normalize}
+
+  body {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",
+    "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans",
+    "Helvetica Neue", sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    scroll-behavior: smooth;
+  }
+
+  #__next {
+    display: flex;
+    min-height: 100vh;
+
+    > * {
+      flex: 1;
+    }
+  }
+
+  *,
+  *::before,
+  *::after
+  {
+  box-sizing: border-box;
+  }
+
+  @media (prefers-reduced-motion: reduce) {
+    * {
+      transition: none !important;
+      animation: none !important;
+      scroll-behavior: auto !important;
+    }
+  }
```

Comment:
suggestion: I wouldn't add these styles to all elements, I prefer the approach of using them only where we need it.